AcadeMeData/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AcademedataConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
Why do you need this one?

Reply:
I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Reply:
> I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Please remove it

Reply:
> I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Please remove it

Reply:
Done